dist/profile/templates/confluence/vhost.conf

```
@@ -2979,6 +2979,8 @@ RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Pipeline\+Nodes\+and\+Processes\+Plugin$" "https://plugins.jenkins.io/workflow-durable-task-step" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Matrix\+Authorization\+Strategy\+Plugin$" "https://plugins.jenkins.io/matrix-auth" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS//Matrix\+Configuration\+Parameter\+Plugin$" "https://plugins.jenkins.io/matrix-combinations-parameter/" [NE,NC,L,QSA,R=301]
```

Need to remove the redundant slash
```suggestion
RewriteRule "^/display/JENKINS/Matrix\+Configuration\+Parameter\+Plugin$" "https://plugins.jenkins.io/matrix-combinations-parameter/" [NE,NC,L,QSA,R=301]
```